### PR TITLE
set default data for "csv to json" and "json to csv"

### LINKED ISSUE #302: [Bug] json to csv - show default data
## Screenshot(s)
Screenshot 2022-02-13 at 2 21 02 AM

Right now, both `csv to json` and `json to csv` apps appear without any data. Its a better experience if the apps appear with some default data which users can change. Why:
- It shows that app is working
- Its easier to modify and play around existing working data, than to write new test data on scratch

## Affected search queries

- csv to json
- json to csv

## Possible solution
The `Component` of the two app, prefill the first column with the default data.

## Your Environment

- Browser information (e.g Chrome v92):
- Operating system: (e.g. mac OS Big Sur)

### PR DESCRIPTION
**Pull request by Sync271**
Closes #302 

## CSV to JSON
image

## JSON to CSV
image

### CODE REVIEW

File: apps/csv-to-json/src/App.jsx [hunk 1]
```diff
@@ -101,7 +101,8 @@ const CopyButton = styled.button`
 //=========================================
 
 function Component() {
-  const [csvData, setCSVData] = useState();
+  const defaultData = "FirstName;LastName\nJohn;Doe"
```
Comment (hargup) [inline apps/csv-to-json/src/App.jsx:104]: @Sync271 I believe we should use `;`, `,`. Rest looks good

Comment (Sync271) [reply apps/csv-to-json/src/App.jsx]: Yeah, my bad!

**Review by hargup — APPROVED** [review]
LGTM

**Comment by hargup** [comment]
> Done :)

Amazing merged 🎉
Thanks for the contribution, looking forward to see what you do next with Felvin 😄 
Feel free to pick up other `good first issues` or propose your own https://github.com/felvin-search/instant-apps/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22

**Pull request merged**